runners/java-fn-execution/src/main/java/org/apache/beam/runners/fnexecution/wire/WireCoders.java

```diff
@@ -38,11 +36,11 @@
    * unknown to the runner are wrapped with length-prefix coders. The inner element coders are kept
```

Comment:
Update comment to state that the components builder is modified to contain the new coder.